mm2src/mm2_main/src/mm2_tests.rs

```diff
@@ -7364,3 +7364,92 @@ fn test_sign_verify_message_eth() {
 
     assert!(response.is_valid);
 }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_no_login() {
+    let coins = json!([
+        {"coin":"RICK","asset":"RICK","rpcport":8923,"txversion":4,"overwintered":1,"protocol":{"type":"UTXO"}},
+        {"coin":"MORTY","asset":"MORTY","rpcport":11608,"txversion":4,"overwintered":1,"protocol":{"type":"UTXO"}}
+    ]);
+
+    let mm = MarketMakerIt::start(
```

Comment:
Please start two nodes here: 
1. A seed node with passphrase, create a couple of orders on it using `setprice`.
2. Light node in "no login" mode. Check that orders data is properly returned by required RPCs.

Please also use `mm2_test_helpers::for_tests::Mm2TestConf` in new tests (you will have to create a new constructor for no login in addition to `seednode` and `light_node`).

Usage example:
https://github.com/KomodoPlatform/atomicDEX-API/blob/ce32ab8da6bd11c880b7517e0f7c746b47fd4699/mm2src/mm2_main/src/mm2_tests/z_coin_tests.rs#L38-L39